--- fedmsg-0.11.1/fedmsg.d/endpoints.py	2014-11-23 03:23:59.491285537 +0200
+++ fedmsg-0.16.4/fedmsg.d/endpoints.py	2016-03-19 15:52:56.357095171 +0200
@@ -25,14 +25,17 @@
         # These are here so your local box can listen to the upstream
         # infrastructure's bus.  Cool, right?  :)
-        "fedora-infrastructure": [
-            "tcp://hub.fedoraproject.org:9940",
+        #"fedora-infrastructure": [
+            #"tcp://hub.fedoraproject.org:9940",
             #"tcp://stg.fedoraproject.org:9940",
-        ],
+        #],
         #"debian-infrastructure": [
         #    "tcp://fedmsg.olasd.eu:9940",
         #],
         #"anitya-public-relay": [
         #    "tcp://release-monitoring.org:9940",
         #],
+        "pld-infra": [
+            "tcp://ep09.pld-linux.org:9940",
+        ],
     },
 )
--- fedmsg-0.11.1/fedmsg.d/base.py~	2014-10-21 18:34:55.000000000 +0300
+++ fedmsg-0.11.1/fedmsg.d/base.py	2014-12-01 21:01:11.065992062 +0200
@@ -20,6 +20,9 @@
 import os
 
 config = dict(
+    # Prefix for the topic of each message sent.
+    topic_prefix="org.pld-linux",
+
     # Set this to dev if you're hacking on fedmsg or an app.
     # Set to stg or prod if running in the Fedora Infrastructure
     environment="dev",
--- fedmsg-0.17.0/fedmsg.d/ircbot.py~	2015-06-24 20:33:04.000000000 +0300
+++ fedmsg-0.17.0/fedmsg.d/ircbot.py	2016-04-26 00:26:38.910365239 +0300
@@ -27,7 +27,7 @@
             timeout=120,
             make_pretty=True,
             make_terse=True,
-            make_short=True,
+            make_short=False,
             line_rate=0.9,
             # Don't show the heartbeat... gross.
             filters=dict(
